modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastFragment.kt

```diff
@@ -926,21 +926,20 @@ class PodcastFragment : BaseFragment(), Toolbar.OnMenuItemClickListener {
         viewLifecycleOwner.lifecycleScope.launch {
             viewModel.refreshState.collect { state ->
                 when (state) {
-                    PodcastViewModel.RefreshState.Error -> {}
                     PodcastViewModel.RefreshState.NotStarted -> {}
                     PodcastViewModel.RefreshState.NewEpisodeFound -> {
                         binding?.swipeRefreshLayout?.isRefreshing = false
-                        (activity as? FragmentHostListener)?.snackBarView()?.let { snackBarView ->
-                            Snackbar.make(snackBarView, getString(LR.string.podcast_refresh_new_episode_found), Snackbar.LENGTH_LONG).show()
-                        }
+                        showSnackBar(getString(LR.string.podcast_refresh_new_episode_found))
+                    }
+                    PodcastViewModel.RefreshState.NoEpisodesFound -> {
+                        binding?.swipeRefreshLayout?.isRefreshing = false
+                        showSnackBar(getString(LR.string.podcast_refresh_no_episodes_found))
                     }
                     is PodcastViewModel.RefreshState.Refreshing -> {
                         if (state.type == PodcastViewModel.RefreshType.PULL_TO_REFRESH) {
                             binding?.swipeRefreshLayout?.isRefreshing = true
                         } else {
-                            (activity as? FragmentHostListener)?.snackBarView()?.let { snackBarView ->
-                                Snackbar.make(snackBarView, getString(LR.string.podcast_refreshing_episode_list), Snackbar.LENGTH_INDEFINITE).show()
-                            }
+                            showSnackBar(getString(LR.string.podcast_refreshing_episode_list), Snackbar.LENGTH_INDEFINITE)
```

Comment:
@geekygecko I reverted your change here to display the snackbar with duration indefinite for when we refresh from the menu options because if the refresh takes more time, the snackbar will hide and the user will not know what happened